src/base_station/debug_console.cpp

```diff
@@ -32,6 +34,12 @@ struct DebugLine
     float r = 0.0f, g = 1.0f, b = 0.0f;
 };
 
+struct Waypoints
```

Comment:
While the combined struct definition and struct variable declaration is clever, its slightly janky. Create a struct type called `Waypoint` that holds a latitude and longitude (they can be floats), and then declare `waypoints` as a variable of type `std::vector<Waypoint>`. This makes more sense because structs are used to group values into a bundle. In this case, the thing that has multiple values (and thus needs bundling) is the waypoint, not the collection of waypoints.